src/devices/Ft4222/DeviceInformation.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Ft4222
+{
+    /// <summary>
+    /// FT4222 device information    
+    /// </summary>
+    public class DeviceInformation
+    {
+        /// <summary>
+        /// Indicates device state.  Can be any combination of the following: FT_FLAGS_OPENED, FT_FLAGS_HISPEED
+        /// </summary>
+        public FtFlag Flags;
```

Comment:
make these properties unless there is some reason to keep them fields

Reply:
Yes, there was a good reason why I didn't use property is because I'm using the members in a call like this:
```csharp
ftStatus = FtFunction.FT_GetDeviceInfoDetail(0, ref flags, ref devInfo.Type, ref devInfo.Id, ref devInfo.LocId, sernum, desc, ref devInfo.FtHandle);
```
Giving the fact that the FT_GetDeviceInfoDetail function is from the external dll, the ref would not work with properties. Now if you have a good way to do this, except creating temporary variables, I'm taking it.
If it's necessary to create properties, I'll go for the temporary variables if nothing better.